app/models/preneeds/service_record.rb

```diff
@@ -3,30 +3,40 @@
 
 module Preneeds
   class ServiceRecord < Preneeds::Base
-    attribute :branch_of_service, String
+    DISCHARGE_TYPES = {
+      'honorable' => '1',
+      'general' => '2',
+      'other' => '7',
+      'bad-conduct' => '5',
+      'dishonorable' => '6',
+      'undesirable' => '4'
+    }.freeze
```

Comment:
Is this being used anywhere? The JSON schema already encodes this field as numeric string values.

Reply:
Good catch @U-DON, its an artifact of the first iteration of Preneeds. I'll remove it in the next PR (attachments).